levels/roles/sem_consulting.md

```diff
@@ -0,0 +1,56 @@
+# Senior Engineering Manager - Consulting focus
+
+## What is Engineering Consulting
+
+(Data) Engineering Consulting focuses on gathering and aligning functional and
+non-functional requirements (elicited from and for the clients) with data
+engineering technologies in order to:
+
+- resolve a client problem,
+- improve the existing landscape, or,
+- enable new capability.
+
+**Note**: space for innovation is not to be disregarded as a 4th possible
+outcome.
+
+## What does a SEM with focus on Engineering Consulting do in the Data Shed
+
+### Commercial Synergy
+
+- participates in Stakeholder (business and tech) meetings after a _lead_ has
+  become an _opportunity_ to assist in gathering and refining requirements.
+- evaluates the requirements on feasibility and fit for the present
+  client/business problem from a techical point of view.
+- assists in compilling commercial agreements including proposals and Statements
+  of Work (SOW) by:
+  - contibuting to listing deliverables,
+  - identifying assumptions and risks, and
+  - estimating effort and cost **with Professional Services**
+- account growth opportunity
+- monitoring and alerting
+
+### Delivery Synergy
+
+- elaborates with the **Delivery Function** on time and resources for the
+  opportunity
+- resourcing _capacity_ and _capability_
+- team formation as a framework
+- early assessment on required skills and levels
+- monitoring and alerting
+- technical refinement and evaluation of proposed solution (SoW)
+- resourcing Capability (skills and levels)
```

Comment:
Nitpicking here but we move 🙌

Lowercase "c"?

Reply:
yeah will squash commits going forward.